src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -499,12 +494,14 @@ public static int calculateInventoryItemsQuantity(CompositePoLine compPOL, List<
     if (compPOL.getCheckinItems() != null && compPOL.getCheckinItems()) return 0;
     switch (compPOL.getOrderFormat()) {
       case P_E_MIX:
-        int quantity = getPhysicalQuantity(locations);
-        return isInventoryUpdateRequiredForEresource(compPOL) ? quantity + getElectronicQuantity(locations) : quantity;
+        int quantity = 0;
+        quantity += isItemsUpdateRequiredForPhysical(compPOL) ? getPhysicalQuantity(locations) : 0;
+        quantity += isItemsUpdateRequiredForEresource(compPOL) ? getElectronicQuantity(locations) : 0;
+        return quantity;
       case PHYSICAL_RESOURCE:
-        return getPhysicalQuantity(locations);
+        return isItemsUpdateRequiredForPhysical(compPOL) ? getPhysicalQuantity(locations) : 0;
       case ELECTRONIC_RESOURCE:
-        return isInventoryUpdateRequiredForEresource(compPOL) ? getElectronicQuantity(locations) : 0;
+        return isItemsUpdateRequiredForEresource(compPOL) ? getElectronicQuantity(locations) : 0;
       case OTHER:
```

Comment:
See my comment below about inventory interactions for "Other" order lines

Reply:
updated items count for "Other" basing on physical quantity.